urllib3/poolmanager.py

```diff
@@ -18,11 +18,6 @@
 __all__ = ['PoolManager', 'ProxyManager', 'proxy_from_url']
 
 
-pool_classes_by_scheme = {
```

Comment:
This is going to be subtly backwards incompatible.

I know there are projects that rely on being able to mutate this to change urllib3's (and as a consequence, requests') behaviour.

You're also removing something that was a module level attribute which is even more backwards incompatible. I have a suggestion:

Let's leave this, and in the `PoolManager` class, let's do something like

```py
if not hasattr(self, 'pool_classes_by_scheme'):
    self.pool_classes_by_scheme = pool_classes_by_scheme
```

This gets you the behaviour you want without breaking other people.

We should also add a note to break this in 2.0